APMrover2/Parameters.cpp

```diff
@@ -161,7 +161,7 @@ const AP_Param::Info Rover::var_info[] = {
     // @Param: PIVOT_TURN_ANGLE
     // @DisplayName: Pivot turn angle
     // @Description: Navigation angle threshold in degrees to switch to pivot steering when SKID_STEER_OUT is 1. This allows you to setup a skid steering rover to turn on the spot in auto mode when the angle it needs to turn it greater than this angle. An angle of zero means to disable pivot turning. Note that you will probably also want to set a low value for WP_RADIUS to get neat turns.
-    // @Units: degrees
```

Comment:
Again I think the longer form of "degrees" is clearer then "deg" but I can go with the majority.